src/bundles/peer-locations.js

```diff
@@ -48,31 +50,39 @@ export default function (opts) {
     'selectPeers',
     'selectPeerLocations',
     'selectBootstrapPeers',
-    (peers, locations = {}, bootstrapPeers) => peers && peers.map(peer => {
-      const peerId = peer.peer.toB58String()
-      const locationObj = locations ? locations[peerId] : null
-      const location = toLocationString(locationObj)
-      const flagCode = locationObj && locationObj.country_code
-      const coordinates = locationObj && [
-        locationObj.longitude,
-        locationObj.latitude
-      ]
-      const connection = parseConnection(peer.addr)
-      const address = peer.addr.toString()
-      const latency = parseLatency(peer.latency)
-      const notes = parseNotes(peer, bootstrapPeers)
-
-      return {
-        peerId,
-        location,
-        flagCode,
-        coordinates,
-        connection,
-        address,
-        latency,
-        notes
-      }
-    })
+    'selectIdentity',
+    (peers, locations = {}, bootstrapPeers, identity) => {
+      const myIP = getPublicIP(identity)
```

Comment:
`myIP` appears to only be used by `isPrivateAndNearby`, and not in this function (except to pass to `isPrivateAndNearby`).

Maybe better to memoize the return value and localize the usage by passing in `identity` to that function and call `getPublicIP` from there instead?

Reply:
Done!